packages/slice-machine/src/legacy/lib/builders/common/Zone/Card/components/Hints/index.tsx

```diff
@@ -20,9 +20,15 @@ const Hint: React.FC<HintProps> = ({
 }) => {
   const fieldPathString = renderHintBase({ item });
 
-  // TODO: Call `swr`'s global `mutate` function when something changes to clear the cache.
```

Comment:
I also tried this solution, but I didn't like it. We have at least two places where we'd have to make this call, and the solution wouldn't be declarative.